src/test.rs

```diff
@@ -428,3 +428,12 @@ fn test_reentrant() {
     sender.send(null.clone()).unwrap();
     assert_eq!(null, receiver.recv().unwrap());
 }
+
+#[test]
+fn test_large_send() {
+    let payload = vec![5u32; 1024 * 1024 * 12];
+    let (tx, rx) = ipc::channel().unwrap();
+    tx.send(payload.clone()).unwrap();
+    let received_payload = rx.recv().unwrap();
```

Comment:
This won't fly on platforms that block while sending large messages. (Including `unix`, as well as the upcoming `windows`.) See existing `big_data()` test...